web/src/tools/testing/bulk_rendering/build.sh

```diff
@@ -31,18 +29,12 @@ builder_describe_outputs \
 
 builder_parse "$@"
 
-if builder_start_action configure; then
-  verify_npm_setup
-  builder_finish_action success configure
-fi
+builder_run_action configure verify_npm_setup
+builder_run_action clean rm -rf ../../../../build/tools/testing/bulk_rendering
 
-if builder_start_action clean; then
-  rm -rf ../../../../build/tools/testing/bulk_rendering
-  builder_finish_action success clean
-fi
-
-if builder_start_action build; then
+do_build ( ) {
   tsc --build "$THIS_SCRIPT_PATH/tsconfig.json" $builder_verbose
   node build-bundler.js
-  builder_finish_action success build
-fi
+}
+
+builder_run_action build do_build
```

Comment:
I'm gonna go ahead and say please reorder functions ahead of builder_run_action statements, and tablify the builder_run_action statements as shown in my comment on sourcemap-root/build.sh